practice/009/answer/impl.ml

```diff
@@ -0,0 +1,9 @@
+let pack list =
+  let rec aux current acc = function
+    | [] -> []    (* Can only be reached if original list is empty *)
+    | [x] -> (x :: current) :: acc
+    | a :: (b :: _ as t) ->
+       if a = b then aux (a :: current) acc t
+       else aux [] ((a :: current) :: acc) t  in
+  List.rev (aux [] [] list);;
+  
```

Comment:
Please add newline at end of the file

Reply:
Actually, here it's delete empty line and finish with newline